gemfiles/rake11.gemfile

```diff
@@ -7,7 +7,3 @@ gem 'rake', '~>11.0', :group => :test
 
 # for CRuby, Rubinius, including Windows and RubyInstaller
 gem "sqlite3", :platform => [:ruby, :mswin, :mingw]
-
```

Comment:
What was the motivation for removing the jruby support?

Reply:
README says it dropped support for JRuby, so I removed them.
If that's my misunderstanding, I'll delete this commit.

Reply:
Ahh maybe I did that when I stopped supporting Rubinius. Don’t worry I’ll resolve it when I merge. Thanks.